tools/cloud-composer-clone/README.md

```diff
@@ -0,0 +1,98 @@
+# Composer clone
```

Comment:
I think it would be better to rename the tool to match what it does, backup and restore. So maybe rename it to Cloud Composer Backup and restore and rename the folder to cloud-composer-backup-restore.